marklogic-data-hub-central/ui/src/util/stomp.ts

```diff
@@ -203,31 +208,44 @@ export class STOMPService {
     // Resolve our Promise to the caller
     this.resolvePromise();
 
+    // for Cypress tests only
+    if ((window as any).Cypress) {
+      (window as any).stompClientConnected = true;
+    }
+
     // Clear callback
     this.resolvePromise = () => {};
   }
 
   // Handle errors from stomp.js
-  public onError = (error: string) => {
-
-    console.error('Error: ' + error);
-
+  public onError = async (error: string, ...other: any[]) => {
+    console.log('STOMP error:', error, other);
```

Comment:
should this be console.error?

Reply:
I changed it to log as it can occur for a range of reasons, like network connection going down. I don't think it merits an error output.

Reply:
Thanks Ryan.